src/Cobol85.tokens

```diff
@@ -0,0 +1,594 @@
+ABORT=1
```

Comment:
generated file
improvement would be to not check this in but build a docker process to build it.